x/stakeibc/client/cli/tx_update_tight_bounds.go

```diff
@@ -0,0 +1,59 @@
+package cli
+
+import (
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/spf13/cobra"
+
+	"github.com/Stride-Labs/stride/v14/x/stakeibc/types"
+)
+
+func CmdUpdateTightBounds() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "update-tight-bounds [chainid] [min-bound] [max-bound]",
+		Short: "Broadcast message update-tight-bounds",
+		Args:  cobra.ExactArgs(2),
+		RunE: func(cmd *cobra.Command, args []string) (err error) {
+			argChainId := args[0]
+			minRedemptionRateStr := args[1]
+			maxRedemptionRateStr := args[2]
+
+			minTightRedemptionRate := sdk.ZeroDec()
+			if minRedemptionRateStr != "" {
+				minTightRedemptionRate, err = sdk.NewDecFromStr(minRedemptionRateStr)
+				if err != nil {
+					return err
+				}
+			}
+			maxTightRedemptionRate := sdk.ZeroDec()
+			if maxRedemptionRateStr != "" {
+				maxTightRedemptionRate, err = sdk.NewDecFromStr(maxRedemptionRateStr)
+				if err != nil {
+					return err
+				}
+			}
```

Comment:
if there's a bad arg, I'm not sure we'd want to pass zero through. might make more sense to change these to
```go
minRedemptionRate := sdk.MustNewDecFromStr(args[1])
maxRedemptionRate := sdk.MustNewDecFromStr(args[2])
```

Reply:
good point, done